core-service/src/main/java/com/crmfoodestablishment/coreservice/entity/Dish.java

```diff
@@ -0,0 +1,29 @@
+package com.crmfoodestablishment.coreservice.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
```

Comment:
Just as open question, is ok @Builder with not complex Classes like that, or it`s as one pleases ?

Reply:
let's remove redundant 'builder'